frontend/ui/elements/menu_keyboard_layout.lua

```diff
@@ -6,12 +6,25 @@ local sub_item_table = {}
 
 for k, _ in orderedPairs(VirtualKeyboard.lang_to_keyboard_layout) do
     table.insert(sub_item_table, {
-        text = Language:getLanguageName(k),
+        text_func = function()
+            local text = Language:getLanguageName(k)
+            if VirtualKeyboard:getKeyboardLayout() == k then
+                text = text .. "   ★"
```

Comment:
Template (for Hebrew no less! ;-))

Reply:
i just copied from the font menu code, for now this will only be the english code for the language that is not translated.